src/components/Node/Node.scss

```diff
@@ -2,9 +2,9 @@
   box-sizing: border-box;
   position: absolute;
   transition: all ease-in-out 0.2s;
-  background-color: #eee;
+  background-color: rgb(250,238,222);
   border: 1px solid lightgrey;
-  border-radius: 5px;
+  border-radius: 3px;
```

Comment:
I guess this can be changed back to 5px

Reply:
I didn't mind to have to img full size actually